.github/workflows/compatibility-tests.yml

```diff
@@ -30,17 +37,21 @@ jobs:
           cache: 'maven'
 
       - name: Compile with default Hazelcast and Hibernate
-        run: ./mvnw compile
+        run: ./mvnw --batch-mode compile
 
       - name: Run tests against HZ ${{ matrix.hazelcast }} and HN ${{ matrix.hibernate }}
-        run: ./mvnw verify -Dmaven.main.skip=true -Dhazelcast.version=${{ matrix.hazelcast }} -Dhibernate.core.version=${{ matrix.hibernate }}
+        run: ./mvnw --batch-mode verify -Dmaven.main.skip=true -Dhazelcast.version=${{ matrix.hazelcast }} -Dhibernate.core.version=${{ matrix.hibernate }}
 
   test-jdks:
     if: github.event_name != 'pull_request' || contains(github.event.pull_request.labels.*.name, 'compatibility-sensitive')
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        hazelcast: [ '5.4.0' ]
```

Comment:
It's not obvious why 5.4.0, WDYT about creating `${{ env.LATEST_HZ_RELEASE }}?

Reply:
Now you are just testing my GHA-fu

16c7de2